airflow/hooks/dbapi.py

```diff
@@ -84,6 +84,7 @@ def __init__(self, *args, schema: Optional[str] = None, **kwargs):
         # from kwargs and store it on its own. We do not run "pop" here as we want to give the
         # Hook deriving from the DBApiHook to still have access to the field in it's constructor
         self.__schema = schema
+        self.log_sql = kwargs.get('log_sql', True)
```

Comment:
Why not default log_sql parameter in constructor - that would also allow you to add docstring.